### DISCUSSION TITLE
Is there a `falsy` check?

### DISCUSSION THREAD

**Discussion by TheColorRed** [slint-ui/slint#6635] [Q&A] [answered]
Is there a `falsy` check or a way to check for a falsy value?

I have a component that has an `icon` input, then an `if` statement. Is there anyway to check to see if a value was assigned to the input and if there was render the component (like I have below):

```slint
component Link inherits HorizontalLayout {
    in property <string> text;
    in property <image> icon;

    if (icon): Image {
        height: 16px;
        width: 16px;
        source: icon;
    }

    Text {
        text: text;
    }
}
```

**Comment by ogoffart** [comment, marked as answer]
I agree we could probably support converting image to bool.
Right now you can do:
```
if (icon.width != 0): Image {
```

**Reply by TheColorRed** [reply]
This works thanks!
